### PR TITLE
Paper and example updates

### PR DESCRIPTION
**Pull request by richford**
- Adds a paper directory with paper.md, etc.
- Updates the examples to get rid of the try/except blocks for Knot creation.
- Updates examples/README.md

### CODE REVIEW

**Comment by richford** [comment]
Merging to get the example updates.

File: doc/index.rst [hunk 1]
```diff
@@ -1,10 +1,58 @@
...
+distributed execution for jobs that fall within the AWS Lambda limits (maximum
+300 seconds of execution time, 1.5 GB of RAM, 512 MB of local storage, and no
+root access). For jobs that require more, we introduce cloudknot to execute
+existing python code on AWS Batch.
+
+Cloudknot takes as input a python function, Dockerizes it for use in an Amazon
+ECS instance, and creates all the necessary AWS Batch constituent resources to
+submit jobs. Users can then use cloudknot to submit and view jobs for a range
+of inputs. For more details, please see the cloudknot
+documentation [@cloudknotdocs].
```
Comment (arokem) [inline doc/index.rst]: The `@cloudknots` should probably be removed?

Comment (arokem) [reply doc/index.rst]: I mean `@cloudknotdocs`

Comment (richford) [reply doc/index.rst]: Thanks. Fixing with #101.

**Pull request merged**